src/helpers/performance-report-analyser.helper.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import chalk from 'chalk';
+
+class PerformanceReportAnalyser {
```

Comment:
In my opinion, this class should take care of analysing ONLY. So it should not display errors, reject or resolv promises ... this is step job, not analyser.

This thing should return an object 
`
{
  requests: [slooooooow_requests_array]
}
`

that's all. The whole thing related to displaying or deciding if it's fine or not, should be inside a step. Moving everything from step implementation to class implementation is not a solution.